index.html

```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
 		  </ul>
 		</section>
 	</main>
+
+	<!-- The footer with the form and social links -->
+	<footer class="footer">
+		<div class="ttl-para">
+			<h2 class="contact-us">contact us</h2>
+			<p class="para ftr-para">
+				If you have an application you are interested in developing, 
+				a feature that you need built or a project that needs coding. I’d love to help with it 
+			</p>
+		</div>
+
+		<!-- The form -->
+		<form  action="https://formspree.io/f/mnqlvaya"	method="POST">
+			<div>
+				<label for="name">Name</label>
+				<input type="text" name="name" class="name name-email" id="name" maxlength="30" required placeholder="Enter your name">
```

Comment:
- [ ] For all input fields, add the `required` attribute to activate the validation on submit

Reply:
done thanks!